service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -82,10 +86,12 @@ public DynamicList buildDynamicListFromHearingDetails(List<Element<HearingBookin
 
     public CaseManagementOrder getCaseManagementOrder(CaseDetails caseDetails) {
         DynamicList list = mapper.convertValue(caseDetails.getData().get("cmoHearingDateList"), DynamicList.class);
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         return CaseManagementOrder.builder()
             .hearingDate(list.getValue().getLabel())
             .id(list.getValue().getCode())
+            .directions(directionHelperService.assignCustomDirections(caseData.getAllParties(), ALL_PARTIES))
```

Comment:
I guess it's correct as we have got all directions customs? Could we add comment flaggin that?

Reply:
What do you think about using this method? I'm a bit concerned as all I want to do is set the assignee. Hence why I copied the majority of this method and removed custom and readOnly props. Is re-use better in this case still?

Reply:
Resolved in conversation.